### ISSUE TITLE
Functional form for the logarithmic limb darkening law is wrong

### ISSUE THREAD

**Issue by JeffValenti** [ExoCTK/ExoCTK#71]
In limb_darkening.ld_profile, the functional form for the logarithm limb darkening law is:

```python
        # Logarithmic
        if name=='logarithmic':
            def profile(m, c1, c2):
                return 1. - c1*(1.-m) - c2*m*(1.-np.log(m))
```

According to [Kreidberg's batman tutorial](https://www.cfa.harvard.edu/~lkreidberg/batman/tutorial.html#limb-darkening-options) (cited in the code), [John Southworth's limb darkening page](http://www.astro.keele.ac.uk/jkt/codes/jktld.html), and [Klinglesmith & Sobieski (1970)](http://adsabs.harvard.edu/abs/1970AJ.....75..175K) (the original paper), the correct functional form is:

```python
        # Logarithmic
        if name=='logarithmic':
            def profile(m, c1, c2):
                return 1. - c1*(1.-m) - c2*m*np.log(m)
```

**Comment by JeffValenti** [comment]
Fixed in pull request #91.

**Comment by hover2pi** [comment]
Merged #94
